tsconfig.json

```diff
@@ -1,23 +1,12 @@
 {
+  "extends": "./node_modules/gts/tsconfig-google.json",
   "compilerOptions": {
-    "target": "es2015",
-    "module": "commonjs",
-    "moduleResolution": "node",
-    "sourceMap": true,
-    "inlineSourceMap": false,
-    "inlineSources": false,
-    "noImplicitAny": false,
-    "suppressExcessPropertyErrors": true,
-    "diagnostics": false,
-    "listFiles": false,
+    "declaration": false,
     "outDir": "dist",
-    "noEmitOnError": false,
-    "strict": true
+    "rootDir": "."
   },
```

Comment:
I compared this with the previous state as well as [TypeScript defaults](https://www.typescriptlang.org/docs/handbook/compiler-options.html), and these are effectively the changes:

* forceConsistentCasingInFileNames
* lib
* noEmitOnError
* noFallthroughCasesInSwitch
* noImplicitReturns
* target

Let me know if there are any concerns.